glide.yaml

```diff
@@ -61,6 +61,10 @@ import:
 - package: github.com/DATA-DOG/godog
   version: v0.6.2
 - package: github.com/elazarl/goproxy
+- package: github.com/gorillalabs/go-powershell
+  version: 3bc7a60b1df80a5766820c4a53d59668f4553f75
+- package: github.com/juju/errors
```

Comment:
@gbraad , are you using this package anywhere?

Reply:
compilation fails if not included, since go-powershell does not do proper dependency management I guess. Glide can not resolve this.

Reply:
Ack